dev-node/11ty+eleventy/11ty+eleventy-0.11.1.ebuild

```
@@ -0,0 +1,77 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit node
+
+DESCRIPTION="Transform a directory of templates into HTML."
+HOMEPAGE="
+	https://www.11ty.dev/
+	https://www.npmjs.com/package/@11ty/eleventy
+"
+
+first="${PN%%+*}"
+second="${PN#*+}"
+SRC_URI="https://registry.npmjs.org/@${first}/${second}/-/${second}-${PV}.tgz -> ${P}.tgz"
```

```suggestion
if [[ ${PN} =~ '+' ]]; then
    PN_FIRST="${PN%%+*}"
    PN_SECOND="${PN#*+}"
    SRC_URI="https://registry.npmjs.org/@${PN_FIRST}/${PN_SECOND}/-/${PN_SECOND}-${PV}.tgz -> ${P}.tgz"
fi
```

could something like that be moved to the eclass instead?